main.py

```diff
@@ -0,0 +1,52 @@
+from fastapi import FastAPI, Depends
+from sqlalchemy.orm import Session
+
+import crud
+import schemas
+from db.database import SessionLocal
+
+app = FastAPI()
+
+
+def get_db() -> Session:
+    db = SessionLocal()
+
+    try:
+        yield db
+    finally:
+        db.close()
+
+
+@app.get("/authors/", response_model=list[schemas.Author])
+def read_authors(db: Session = Depends(get_db)):
+    return crud.get_all_authors(db)
+
+
+@app.get("/authors/{author_id}", response_model=schemas.Author)
+def read_author_by_id(author_id: int, db: Session = Depends(get_db)):
+    return crud.get_author_by_id(db, author_id)
+
+
+@app.post("/authors/create/", response_model=schemas.AuthorCreate)
+def create_author(author: schemas.AuthorCreate, db: Session = Depends(get_db)):
+    return crud.create_author(db, author)
+
+
+@app.get("/books/", response_model=list[schemas.Book])
+def read_books(db: Session = Depends(get_db)):
+    return crud.get_all_books(db)
+
+
+@app.get("/books/{book_id}", response_model=schemas.Book)
```

Comment:
```suggestion
@app.get("/books/{book_id}/", response_model=schemas.Book)
```
the same in other similar cases